### ISSUE TITLE
Does edireader has builtin JSON output option?

### ISSUE THREAD

**Issue by sfc-gh-nakincilar** [BerryWorksSoftware/edireader#19]
Is there any sample code or project to output as JSON?

**Comment by canabrook** [comment]
Yes, there is an EDI to JSON facility built upon the core EDIReader parser, producing JSON output from EDI input. Take a look https://github.com/BerryWorksSoftware/edi-json <https://github.com/BerryWorksSoftware/edi-json> for samples of the JSON and description of features. Let me know if you have further questions, and I will be pleased to provide answers. We can arrange a phone conversation if you like.

Scott
